CHANGELOG.md

```diff
@@ -13,6 +13,51 @@ Branch: `nilrt/master/hardknott`
 
 ----
 
+## 9.1
+Branch: `nilrt/22.8/hardknott`
+
+### nilrt
+
+#### Added
+- Added NILRT GRUB version 22.8 to dist feed
+- Added scripts to diff feeds between NILRT releases
+- Added configuration to use an NI-internal hashserver for builds
```

Comment:
This is experimental. We should probably leave it out of the changelog, lest someone thinks that it is permanent infrastructure.